qcfractal/storage_sockets/models/collections_models.py

```diff
@@ -0,0 +1,237 @@
+import datetime
+from sqlalchemy import (Column, Integer, String, DateTime, Boolean, ForeignKey, JSON,
+                        Enum, Float, Binary, Table, ARRAY,
+                        inspect, Index, UniqueConstraint)
+from sqlalchemy.orm import relationship, column_property
+from sqlalchemy import select, func, tuple_, text, cast
+from qcfractal.storage_sockets.models import Base
+from qcfractal.storage_sockets.models import MoleculeORM
+from sqlalchemy.dialects.postgresql import array_agg
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql.functions import GenericFunction
+from sqlalchemy.ext.hybrid import hybrid_property
+
+
+# class json_agg(GenericFunction):
+#     type = postgresql.JSON
+
+class json_build_object(GenericFunction):
```

Comment:
Is it common for these classes to be snake case?

Reply:
It's an unusual case, where there construct a method (json_agg) using classes (GenericFunction).
In the code it's used as a method.